src/test/java/picard/util/VcfTestUtils.java

```diff
@@ -0,0 +1,116 @@
+package picard.util;
```

Comment:
There already is a VcfTestUtils in picard.vcf.  I think it overlaps in content with this.  FYI - I believe that picard-private couldn't access picard (public)'s version, so that's why there's duplication.